config/install/field.storage.node.field_activity_color.yml

```
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - colorapi
+    - node
+id: node.field_activity_color
+field_name: field_activity_color
+entity_type: node
+type: colorapi_color_field
+settings: {  }
+module: colorapi
+locked: false
+cardinality: 1
+translatable: true
+indexes: {  }
+persist_with_no_fields: false
```

```suggestion
persist_with_no_fields: true
```

for safer upgrade path